platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -505,4 +585,79 @@ protected void configureTimeouts(
   public void addOutInterceptors(Interceptor<? extends Message> inteceptor) {
     this.clientFactory.getOutInterceptors().add(inteceptor);
   }
+
+  private SSLSocketFactory getSSLSocketFactory(
+      String sslProtocol, String alias, KeyManager[] keyManagers, TrustManager[] trustManagers)
+      throws KeyManagementException, NoSuchAlgorithmException {
+
+    for (int i = 0; i < keyManagers.length; i++) {
+      if (keyManagers[i] instanceof X509KeyManager) {
+        keyManagers[i] = new AliasSelectorKeyManager((X509KeyManager) keyManagers[i], alias);
+      }
+    }
+
+    SSLContext context = SSLContext.getInstance(sslProtocol);
```

Comment:
Can we just add a check in here to ensure that whatever is passed in is part of the list in the system property `jdk.tls.client.protocols` so that way no one can drop this to a less secure protocol and go around the settings that we try to use for the rest of the JVM.

Reply:
Fixed.